sql/core/src/main/scala/org/apache/spark/sql/scripting/SqlScriptingInterpreter.scala

```diff
@@ -58,44 +61,108 @@ case class SqlScriptingInterpreter() {
       case _ => None
     }
 
+  private def transformBodyIntoExec(
```

Comment:
nit: doc comment

Reply:
let's also use `transformBodyIntoExecutable`, to be consistent with the other function

Reply:
Done.